### PR TITLE
Update doc to latest version & adjust styling

### PR DESCRIPTION
**Pull request by NaughtyChas**

### CODE REVIEW

File: docs/handbook/index.md [hunk 1]
```diff
@@ -41,15 +41,16 @@ title: 参赛手册
 
 ### 2.1 报名
 
-1. 原则上，玩家在**报名时**须满足如下条件，方可获得 OFFC cirno 的**参与资格**：
+1. 具有**参赛资格**的玩家，只要在报名期间内如实填写报名表，在通过审查后即可参加OFFC cirno。所谓**参赛资格**指的是如下条件：
```
Comment (CloneWith) [inline docs/handbook/index.md:44]:
```suggestion
1. 具有**参赛资格**的玩家，只要在报名期间内如实填写报名表，在通过审查后即可参加OFFC cirno。**参赛资格**所指具体条件如下：
```

File: docs/handbook/index.md [hunk 2]
```diff
@@ -41,15 +41,16 @@ title: 参赛手册
...
    - **报名结束时**，osu!std 模式表现分（PP，Performance Point）在 <Highlight color="#198dff">**4700-6500**</Highlight> 范围内。
    - **报名结束时，ETX≤6.4**。
    - 无小号、代打、外挂等相关情况。账号无禁言以外的不良记录。
    - 能够使用中文交流，能够使用 QQ 或 Discord 进行即时通讯。
    - 在 2024.7.1 至 2024.8.30 有一定的 osu!Standard 游玩记录。
-2. 若玩家不满足 2.1.1 的 PP 要求，但其 PP 在 <Highlight color="#8f3fcf">**4500-6700**</Highlight> 范围内且满足所有其他要求，则该玩家仍可先报名。报名后，该玩家将进入<Highlight color="#8f3fcf">待定区</Highlight>。在总报名人数未达到预期人数时，OFFC 的工作人员会视实际情况，按如下顺序允许待定区的选手参与比赛：
-   - 报名时间较早者优先，时间精确到分钟；
-   - 报名时间相同的，报名时 PP 更接近 4700 或 6500 者优先。
+2. 若玩家**报名结束时**不满足 2.1.1 的 pp 要求，但其 pp 在 <Highlight color="#8f3fcf">**4500-6800**</Highlight><sub>**原6700，据8月20日群公告调整为6800**</sub>  范围内且满足所有其他要求，则该玩家仍可先报名。报名后，该玩家将进入<Highlight color="#8f3fcf">待定区</Highlight>。在总报名人数未达到预期人数时，OFFC 的工作人员会视实际情况，按如下顺序允许待定区的选手参与比赛：
```
Comment (CloneWith) [inline docs/handbook/index.md:50]:
```suggestion
2. 若玩家**报名结束时**不满足 2.1.1 的 pp 要求，但其 pp 在 <Highlight color="#8f3fcf">**4500-6800**</Highlight><sub>**（原6700，据8月20日群公告调整为6800**）</sub>范围内且满足所有其他要求，则该玩家仍可先报名。报名后，该玩家将进入<Highlight color="#8f3fcf">待定区</Highlight>。在总报名人数未达到预期人数时，OFFC 的工作人员会视实际情况，按如下顺序允许待定区的选手参与比赛：
```

File: docs/handbook/index.md [hunk 3]
```diff
@@ -316,31 +319,31 @@ title: 参赛手册
...
-   - 举例：在一场比赛中，红队在Ban图时用尽了默认时间，随后消耗了30秒的补充时间才完成Ban图。那么，在红队选图时，其可以利用的时间是120秒（默认）+60秒（一类补充）。若此后红队又消耗了30秒的补充时间，则其Pick时可以利用的时间变为120秒（默认）+30秒（一类补充）。
-3. 如 [6-2.8](#6-2-比赛流程) 所述，双方上人的默认时限为<FntColor color="#198dff">**120秒**</FntColor>（常规阶段）或<FntColor color="#7d75ee">**180秒**</FntColor>（EX阶段）。若在时间用尽后，任意一方队伍仍未上齐选手或选手的Mod不正确，则裁判将给出警告，并开始消耗该队伍的<FntColor color="#ffa500">二类补充时间</FntColor>。二类补充时间共有<Highlight color="#ffa500">**90秒**</Highlight>，且具有全局性。
-   - 举例：在一场比赛中，红队在第一次上人时用尽了默认时间，随后消耗了30秒的补充时间才完成上人。那么，在下一张图上人时，其可以利用的时间是120秒（默认）+60秒（二类补充）。若此后红队又消耗了30秒的补充时间，且进入EX阶段，则其上人时可以利用的时间变为180秒（EX默认）+30秒（二类补充）。
-4. 一类补充时间用尽后，在默认时间用尽的情况下，视正在进行的行动判断处理方法。
+1. <FntColor color="#27efcb">Protect</FntColor>、<FntColor color="#232323">Ban</FntColor>、<FntColor color="#198dff">Pick</FntColor>、<FntColor color="#7d75ee">Trap</FntColor>、<FntColor color="#ffbf05">上人</FntColor>，是每支队伍需要进行的五项行动。这五项行动均有对应的默认时限，该默认时限每回合重新计算。此外，每支队伍还拥有一定的补充时间。补充时间在每场比赛中具有全局性，即在该场比赛中不会恢复、不会重新计算。<FntColor color="#9a38d7">一类补充时间</FntColor>和<FntColor color="#ffa500">二类补充时间</FntColor>单独计算、互不影响。
```
Comment (CloneWith) [inline docs/handbook/index.md:326]: 图片

The appearance of `Ban` isn't so good on dark mode. Consider using `Highlight`.

File: docs/handbook/index.md [hunk 4]
```diff
@@ -408,8 +412,9 @@ title: 参赛手册
 
 1. 感谢 <Highlight color="#232323">**DESU.Life**</Highlight> 为冠军队伍赞助猫盘！欢迎访问他们的[官网](https://desu.life/#meowpad)和[旗舰店](https://kagamistudio.taobao.com)，了解关于猫盘的更多信息！
 2. 感谢[My Angel Kuroe](https://osu.ppy.sh/users/13217389)为SVP赞助碧蓝档案设定集！
+3. 感谢[Dr_Shindo](https://osu.ppy.sh/users/30895664)为TVP赞助osu!主题马克杯！
 3. 如果您愿意赞助OFFC，欢迎通过QQ联系赛事主办kitakore（<Highlight color="#198dff">**273772334**</Highlight>）。OFFC Staff感谢您对本赛事的支持！
 
-<img src="/img/OFFCstaff.png" width={500} alt="Staff card" /> <img src="/img/sponsor.png" class="auto-invert" width={200} alt="Sponsor logo" />
+<img src="/img/OFFCstaff.png" width={500} /> <img src="/img/sponsor.png" width={200} />
```
Comment (CloneWith) [inline docs/handbook/index.md:418]: Is this change intentional?

File: docs/handbook/index.md (hunk 3 above)
Comment (NaughtyChas) [reply docs/handbook/index.md]: Decided to use bold font instead. See commit [bd6e19d](https://github.com/offfc/offfc.github.io/pull/38/commits/bd6e19d8368d6435a9cf4ca85468a32364b01c27).

File: docs/handbook/index.md (hunk 4 above)
Comment (NaughtyChas) [reply docs/handbook/index.md]: Seems like there is something wrong when resolving conflicts when syncing fork to the main. Added this line back in commit [bd6e19d](https://github.com/offfc/offfc.github.io/pull/38/commits/bd6e19d8368d6435a9cf4ca85468a32364b01c27).

**Review by CloneWith — APPROVED** [review]
Look good to merge. Let's do it!
